UnitTests/GitCommands.Tests/Git/Gpg/GitGpgControllerTests.cs

```diff
@@ -161,9 +161,6 @@ public void Validate_GetTagVerifyMessage(int usefulTagRefNumber, string expected
                         var gitRef = new GitRef(_module, objectId, "refs/tags/TagName");
                         revision.Refs = new[] { gitRef };
 
-                        var args = new GitArgumentBuilder("verify-tag") { gitRef.LocalName };
-                        validate = _executable.StageOutput(args.ToString(), "");
```

Comment:
📝 I'm not sure what this was testing, so not sure what impact removing it has

Reply:
![image](https://user-images.githubusercontent.com/4403806/116398821-8e760b80-a86b-11eb-9039-b387ebaa406d.png)

Looks a combination of a) badly arranged test - for non-deref git command isn't invoked, and b) update in NUnit, that now starts asserting where previously it wouldn't.